src/ripple/core/impl/JobQueue.cpp

```diff
@@ -161,10 +162,18 @@ JobQueue::setThreadCount (int c, bool const standaloneMode)
     else if (c == 0)
     {
         c = static_cast<int>(std::thread::hardware_concurrency());
-        c = 2 + std::min (c, 4); // I/O will bottleneck
-
-        JLOG(m_journal.info()) << "Auto-tuning to " << c <<
-                            " validation/transaction/proposal threads";
+        if (validator_)
+            c = 2 + std::min(c, 4); // I/O will bottleneck
```

Comment:
I would actually have validators and client handlers get the same treatment, but that's controversial. For this PR, I was thinking to not affect validators by default.